extension/chrome/settings/modules/compatibility.htm

```diff
@@ -16,6 +16,14 @@ <h1>OpenPGP key compatibility test</h1>
 
       <div class="line">Locally test any public or private key.</div>
 
+      <div class="line">
+        <label
+          >Search for public key on FlowCrypt Attester
```

Comment:
Since this uses PubLookup, which is indeed the better choice, it is looking up public keys on other places as well. It would be good to change this to `Search for public key on key servers`

Further, this page is now probably in need of some instruction on top. "You may use this page to ..." and "You can either paste the public key you want to check, or you can have one loaded from key servers by entering the email" and similar.